website/content/docs/security/acl/acl-roles.mdx

```diff
@@ -0,0 +1,378 @@
+
+## Workflow Overview
+
+Roles are configurations link several policies to a token. The following procedure describes the workflow for implementing roles.
+
+1. Assemble rules into policies (see [Policies](/docs/security/acl/acl-policies)) and register them in Consul.
+1. Define a role and include the policy IDs or names.
+1. Register the role in Consule and link it to a token.
+1. Distribute the tokens to users for implementation.
+
+## Creating Roles
+
+Creating roles is commonly the responsibility of the Consul ACLs administrator.
+Roles have several attributes, including service identities and node identities.
+Refer to the following documentation for details:
+
+- [Role Attributes](#role-attributes)
+- [Service Identities](#service-identities)
+- [Node Identities](#node-identities)
+
+Use the Consul command line or API endpoint to create roles.
+
+### Command Line
+
+Issue the `consul acl role create` command to create roles. In the following example, a role named `crawler` is created that contains a policy named `crawler-kv` and a policy named `crawler-key`.
+
+```shell-session
+$ consul acl role create -name "crawler" -description "web crawler role" -policy-name "crawler-kv" -policy-name "crawler-key"
+```
+
+Refer to the [command line documentation](/command/acl/role) for details.
+
+### API
+
+Make a `PUT` call to the `acl/role` endpoint and specify the role configuration in the payload to create roles. You can save the role definition in a JSON file or use escaped JSON in the call. In the following example call, the payload is defined externally.
+
+```shell-session
+$ curl -X PUT --data @payload.json http://127.0.0.1:8500/v1/acl/role
+```
+
+Refer to the [API documentation](/api-docs/acl/roles) for details.
+
+## Role Attributes
+
+Roles may contain the following table describe the attributes:
+
+- `ID`: The `ID` is an auto-generated public identifier. You can specify the role `ID` when linking it to tokens.
+- `Name`: A unique meaningful name for the role. You can specify the role `Name` when linking it to tokens.
+- `Description`: (Optional) A human-readable description of the role.
+- `Policies`: Specifies a the list of policies that are applicable for the role. The object can reference the policy `ID` or `Name` attribute.
+- `ServiceIdentities`: Specifies a list of services that are applicable for the role. See [Service Identities](#service-identities) for details.
+- `NodeIdentities`: Specifies a list of nodes that are applicable for the role. See [Node Identities](#node-identities) for details.
+- `Namespace`: <EnterpriseAlert inline /> The namespace this policy resides within. Roles can only be linked to policies that are defined in the same namespace and admin partition. See [Namespaces](/docs/enterprise/namespaces) and [Admin Partitions](/docs/enterprise/admin-partitions) for additional information. Requires Consul Enterprise 1.7.0+.
+
+## Service Identities
```

Comment:
Related to the other suggestion about these "x identities", these are more about policies, not roles, so I think these sections (Service Identities and Node Identities) should be part on the policies page.